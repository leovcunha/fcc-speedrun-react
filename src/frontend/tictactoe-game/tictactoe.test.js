import React from "react";
import renderer from "react-test-renderer";
import TicTacToe from "../tictactoe-game";

describe("tictactoe", () => {
    it("renders without crashing", () => {
        const component = renderer.create(<TicTacToe/>).toJSON();
        expect(component).toMatchSnapshot();
    });
});
