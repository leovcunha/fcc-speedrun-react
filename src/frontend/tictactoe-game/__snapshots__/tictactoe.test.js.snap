// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tictactoe renders without crashing 1`] = `
<div
  className="tictactoe jumbotron"
>
  <div
    className="row h1 justify-content-center"
  >
    Tic Tac Toe
  </div>
  <hr
    className="p-5"
  />
  <div>
    <div
      className="buttons"
    >
      <div
        className="row justify-content-center mb-5"
      >
        <h2>
          Choose your Mark!
        </h2>
      </div>
      <div
        className="row justify-content-center"
      >
        <button
          className="btn btn-primary mr-2"
          onClick={[Function]}
        >
          X
        </button>
        <button
          className="btn btn-primary ml-2"
          onClick={[Function]}
        >
          O
        </button>
      </div>
    </div>
  </div>
</div>
`;
